repository-postgresql/src/test/java/org/lfenergy/compas/scl/data/repository/postgresql/CompasSclDataPostgreSQLRepositoryTest.java

```diff
@@ -22,4 +28,18 @@ protected CompasSclDataRepository getRepository() {
     void beforeEach() {
         repository = new CompasSclDataPostgreSQLRepository(PostgreSQLServerJUnitExtension.getDataSource());
     }
+
+    /**
```

Comment:
Don't think you should make it Java Doc, but just a comment starting with // or /* */